### DISCUSSION TITLE
The page for a specific case should contain information on which topic it belongs to

### DISCUSSION THREAD

**Discussion by augusto-herrmann** [tosdr/edit.tosdr.org#1065] [Ideas] [answered]
If you are navigating through a specific case page, it is currently a hassle to find out which topic it belongs to. There should be the name of the topic and a link to it on this page.

Example: [Case 178](https://edit.tosdr.org/cases/178) belongs to [topic 40: User logs](https://edit.tosdr.org/topics/40), but there is no link (or even text!) pointing from the former to the latter.

**Comment by augusto-herrmann** [comment]
You think we need to discuss this first, @JustinBack ?

I can't see any disadvantage in doing so. Since we're talking about a single case page, it shouldn't burden the server much.

**Reply by JustinBack** [reply]
Sounds like a reasonable change and I'm not sure why this wasn't done before.


I have moved this to a github discussion from the issue and created the roadmap entry.

https://git.tosdr.org/tosdr/roadmap/projects/1

https://git.tosdr.org/tosdr/roadmap/issues/1

**Comment by JustinBack** [comment, marked as answer]
Roadmap link (for selecting this as an answer)

https://git.tosdr.org/tosdr/roadmap/projects/1

https://git.tosdr.org/tosdr/roadmap/issues/1
